src/main/java/school/lemon/changerequest/java/introduction/hw1/Task4.java

```diff
@@ -4,6 +4,14 @@
 public class Task4 {
 
     public static double getSumOfGreatest(double a, double b, double c) {
-        return 0;
+
+        double sum = 0;
```

Comment:
In such case there is no need to initialize sum with any value (as you'll calculate sum in any cases)